frontend/src/metabase-lib/lib/Question.js

```diff
@@ -231,17 +231,28 @@ export default class Question {
     }
 
     alertType() {
-        const mode = this.mode();
         const display = this.display();
 
         if (!this.canRun()) {
             return null;
         }
 
+        const isLineAreaBar = display === "line" || display === "area" || display === "bar"
+
         if (display === "progress") {
             return ALERT_TYPE_PROGRESS_BAR_GOAL;
-        } else if (mode && mode.name() === "timeseries") {
+        } else if (isLineAreaBar) {
             const vizSettings = this.card().visualization_settings;
+
+            // Don't support multiseries questions
+            // FIXME Atte Keinänen 1/18/17: This only detects visualizations where the user has manually
```

Comment:
Is this something that you're intending to fix in this PR? I don't really know what the context of the FIXME is and what is missing if it's not there.